source/docs/hardware-components/fasteners-guide.rst

```diff
@@ -0,0 +1,182 @@
+..    include:: <isonum.txt>
+
+Fasteners Guide
+===============
+If you want your robot to stay in one piece, you will need fasteners. Fasteners are any pieces of hardware that mechanically join parts. This guide will discuss common types of fasteners, and how to use them on your robot.
+
+Threaded Fasteners
+------------------
+Bolts, screws and nuts are all threaded fasteners -- meaning they use a helical thread to join parts.
+
+Thread Basics
+^^^^^^^^^^^^^
+To understand threaded fasteners, you will need to know some basic terminology
+
+
+.. figure:: images/thread-diagram.png
+   :alt: diagram showing thread terminology
+
+External thread
+    A thread on an outside surface. Bolts have external threads
+Internal thread
+    A thread on an inside surface. Nuts have internal threads
+Thread pitch
+    The distance from one thread to another, measured along the length of the thread. This can also be expressed as the number of threads per unit of length, for example threads per inch
+Major diameter
+    The larger of the two diameters giving the height of the thread profile.
+Minor diameter
+    The smaller of the two diameters giving the height of the thread profile.
+
+Thread Standards and Naming
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+There are 2 main systems of naming and sizing threads that you will encounter in FTC: ISO metric sizing and UTS imperial sizing.
```

Comment:
```suggestion
There are two main systems of naming and sizing threads that you will encounter in FTC: ISO metric sizing and UTS imperial sizing.
```
The order here should match the order of the subsections; either reverse the order of the two standards in the sentence or swap the two sections.